src_py/apiServer/globalVars.py

```diff
@@ -15,48 +16,39 @@
 
 lossMaps = []
 
+# Get the components of the current system:
 ARCHITECTURE_INDEX = 4
 GRAPH_INDEX = 5
-# Get the components of the current system:
-
-E_TRAINING = 0
-E_PREDICTION = 1
 
 username = os.getlogin()
 jsonPathLocation = '/home/{}/workspace/NErlNet/jsonPath'.format(username)
```

Comment:
Use the new path location: "/usr/local/lib/nerlnet-lib/NErlNet" that should be the same across all platforms after using the script NerlnetInstall.sh 
https://github.com/leondavi/NErlNet/blob/master/NerlnetInstall.sh 

@dordor7 @kapelnik @evgenyan95